tsfile/src/main/java/org/apache/iotdb/tsfile/utils/BloomFilter.java

```diff
@@ -160,7 +160,12 @@ public int hashCode() {
     }
 
     public int hash(String value) {
-      return Math.abs(Murmur128Hash.hash(value, seed)) % cap;
+      int res = Murmur128Hash.hash(value, seed);
+      if (res == Integer.MIN_VALUE) {
+        res = 0;
+      }
+
+      return Math.abs(res) % cap;
```

Comment:
Will this cause incompatible with 0.13 tsfile?
If this case will cause error, add a UT

Reply:
This compatible with 0.13 tsfile. Math.abs(Integer.MIN_VALUE) will become 0 rather than Integer.MIN_VALUE. Integer.MIN_VALUE will cause array out of bound error in before version